docs/content/en/main/concepts/investigations/edge-multi-cluster copy.md

```diff
@@ -0,0 +1,135 @@
+   Multi-cluster use case.
+
+A key area of investigation for `kcp` is exploring large-scale deployment of workloads that can operate under constraints and
+ conditions unlike those accomodated by upstream Kubernetes. Ideally, we want declaritively defined workloads to operate across a
+  hierarchy combining KCP workspaces and Kubernetes cluster namespaces to achieve greater scale and control over placement. Deployment
+   of workloads across many thousands of destinations that include namespaced and non-namespaced objects, receiving state and reporting
+    status (grouped and individually), require new interfaces based on (and extending) KCP primitives.
+
+## Goal: Placement strategies
+
+There are many different ways to characterize deployments of workloads in the Edge use case. Support for blue/green, canary, carbon
+ intensity sensitive, and threat avoidance are some of the deployment strategies to consider when casting out 100's or 1000's of
+  workloads to millions of destinations. Having the ability to assign workloads using a declaritively defined placement strategy to
+   edge cluster locations, even if they are offline at times, is a key requirement of the edge multi-cluster use case.
+
+## Overlap of roles, responsibilities, integrations, and security
+
+Today's smartphone is not only the property of the user who purchased it. The reality is that application developers (app store),
+ corporate entities (vpn profiles), and the smartphone manufacturer have roles and responsibilities on these devices. Edge cluster
+  locations are no different in this respect. A scalable Edge multi-cluster solution can thrive and advance if there is support for
+   community contribution powered by integration and proper security features. All of these considerations must work across an evolving
+    hierarchy that combines small and large control plane API services.
+
+## Hierarchy and Scale are core considerations
+
+KCP has introduced the concept of transparent multi-cluster which can be used to expand the number of entry points for workloads and
+ operations by reducing the size of the control plane to a single binary. Synchronization between KCP virtual workspaces and physical
+  clusters has made it possible to deploy workloads centrally and then fan out to any defined subset of clusters. We would like to
+   leverage multiple levels of hierarchy (cascading multiple instances of KCP and Kubernetes cluster) to arrive at a distributed system
+    capable of reaching any targeted edge cluster with/without edge devices attached. Building out such a hierarchy will require a
+     scalable database to hold the objects referenced by each of their associated API servers. The default database for KCP and
+      Kubernetes is etcd. We will investigate alternatives to etcd distribution and replacement databases to increase scalability with
+       sacrificing performance and resilience without negatively impacting cost.
+
+## Goal: Modularity is key
```

Comment:
The heading of this section is motherhood and apple pie, the first sentence is a vague salute, and the remainder makes a different point.

I do not think we need to advocate modularity as a vague generality.  I think we *should* make it clear somewhere --- probably at the top of the document --- that we are talking about a peer of TMC, those being two layers that rest on the other two lower layers of kcp.

Regarding flexibility of hierarchy: I think we should clearly say that the customer brings a hierarchy that is connected with their business, we do not get to dictate the hierarchy.  Of course we can impose constraints based on what we consider feasible, but these should be no more than really necessary.

Regarding flexibility in deployment and integration: perhaps you are saying that the customer brings their own inventory management solution.  If so, I think that is a better way to say it.  Perhaps you are trying to get at another distinction from TMC, which I still have not found a clear formulation for, but it is about the fact that a customer doing edge computing needs a solution not only for containerized workload but also lower layers of the stack, down to stuff described with words like "infrastructure".

Reply:
removed the modularity goal altogether.  It is inherent of Kube and we are not changing this, so why mention it.